config.example.yaml

```diff
@@ -190,6 +190,17 @@ templates:
           /{api:sys}:
             x-subspec: *wp/sys/1.0.0
 
+# Should be used instead of wp-default-1.0.0 for private wiki domains
+  wp-secure-1.0.0: &wp/secure/1.0.0
+    x-subspecs:
+      - paths:
+          /{api:v1}:
+            x-subspec: *wp/content/1.0.0
+            security:
```

Comment:
According to the swagger spec, the `security` stanza should contain one or more [securityRequirementObject](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#securityRequirementObject)s.

So, perhaps something like this?

```yaml
security:
  - mediawiki_auth:
    - read
```